tests/test_adjust_all_quanta.py

```diff
@@ -111,13 +110,9 @@ class AdjustAllQuantaTestCase(unittest.TestCase):
 
     @staticmethod
     def make_butler() -> Butler:
-        try:
-            DATA_DIR = os.path.join(getPackageDir("daf_butler"), "tests", "data", "registry")
```

Comment:
I had no idea we were doing this. The test data is also not available in a pipelines docker container either (we strip the tests directories to make the container smaller)